streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -238,13 +243,22 @@ private void init(final StateStore root) {
         updateBufferMetrics();
         open = true;
         partition = context.taskId().partition();
+        consistencyEnabled = StreamsConfig.InternalConfig.getBoolean(
+            context.appConfigs(),
+            IQ_CONSISTENCY_OFFSET_VECTOR_ENABLED,
+            false);
     }
 
     @Override
     public boolean isOpen() {
         return open;
     }
 
+    @Override
+    public Position getPosition() {
+        return position;
```

Comment:
This isn't harmful, but it will also never be used, since this store is not exposed to IQ. I'm not sure if we still want this implementation or if we'd rather just throw an `UnsupportedOperationException`. I think that's something you can do in a follow-up PR if you like.